packages/compass-workspaces/src/stores/workspaces.spec.ts

```diff
@@ -73,7 +73,11 @@ describe('tabs behavior', function () {
       const store = configureStore();
       openTabs(store);
       store.dispatch(
-        openWorkspace({ type: 'Collection', namespace: 'test.bar' })
+        openWorkspace({
+          type: 'Collection',
+          namespace: 'test.bar',
+          subTab: 'Indexes',
```

Comment:
This doesn't look right, this should be optional when opening a workspace, you can't provide a required subTab and pass an initial query / agg at the same time

Reply:
We use `useWorkspace` hook to expose `openCollectionWorkspace` and `openEditViewWorkspace`. These methods now contain the logic to decide which subtab to open, which was previously in `collection-tab.ts`. With that, I think it should be required for this method and optional for `openCollectionWorkspace`.